requirements.txt

```diff
@@ -1,23 +1,3 @@
-Adafruit-GPIO==1.0.1
```

Comment:
So a lot of these got removed, because they were added from a `pip freeze > requirements.txt`, but most of these are dependencies of the main libraries we have. To keep it small, only add the things _we_ need, and remove the other junk.